.github/workflows/CI.yml

```diff
@@ -12,11 +12,9 @@ jobs:
   Darwin:
     name: Darwin
     runs-on: macos-latest
-    strategy:
-      matrix:
-        xcode: ['/Applications/Xcode_11.3.app/Contents/Developer']
```

Comment:
You can't reuse this?

Reply:
I think we don't really need a matrix here ... we just need to specify environment

Reply:
I meant more as a reusable element. Not sure what a strategy matrix is, I haven't looked into GitHub workflows yet.

Reply:
Yeah, I just change this